ui/src/app/pages/workspace/workspace-edit.tsx

```diff
@@ -604,9 +603,12 @@ export const WorkspaceEdit = fp.flow(withRouteConfigData(), withCurrentWorkspace
 
     get isOtherSpecificPopulationValid() {
       const researchPurpose = this.state.workspace.researchPurpose;
-      return this.isSpecificPopulationValid && (
-          !fp.includes(SpecificPopulationEnum.OTHER, researchPurpose.populationDetails) ||
-          (researchPurpose.otherPopulationDetails && researchPurpose.otherPopulationDetails.length <= 100));
+      // Avoids null pointers. We don't want to show this error message if nothing is selected yet.
```

Comment:
I noticed previously that this error would show up if other wasn't selected, if `isSpecificPopulationValid` was false (e.g., nothing was yet selected). This fixes that.

Reply:
thanks for this fix!